doc/example/limitations_non_virtual_method.cpp

```diff
@@ -25,3 +24,10 @@ MOCK_BASE_CLASS( mock_base, base )
     MOCK_METHOD( method, 0 )
 };
 //]
+
+BOOST_AUTO_TEST_CASE(method_not_called_through_base)
+{
+    mock_base b;
+    MOCK_EXPECT(b.method).never();
+    static_cast<base*>(&b)->method(); // Doesn't call the mocked method
```

Comment:
nit: I think the comment is not needed given the automatic documentation provided by the previous line 😁

Reply:
I extended the comment a bit. I think it is still important as it should show a very specific point